open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def mkframe(country):
</pre><p>why not add questions for this function, and add tests?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<ol><li><strong>I don‘t understand</strong> Let's now switch to investigating a specific country. Let's create a frame that contains the data on infections indexed by date , this message, isn't it a question for people do the code below ?</li><li>Tests, there some problems......sorry</li></ol>